standard/docs/en/schema/merging.md

```diff
@@ -51,106 +52,163 @@ If the record is complete with versioning information, then the versioning secti
 
 .. jsoninclude:: ../examples/merging/merge-award-2.json
    :jsonpointer: /releases
-   :expand: releases, award
+   :expand: releases, tag, awards
    :title: awardTwo
 
 ```
 
 ```eval_rst
 
 .. jsoninclude:: ../examples/merging/merged.json
-   :jsonpointer: 
-   :expand: records, compiledRelease, tender, award, tag
+   :jsonpointer:
+   :expand: records, compiledRelease, tag, tender, awards
    :title: record
 
 ```
 
 ```eval_rst
 
 .. jsoninclude:: ../examples/merging/versioned.json
-   :jsonpointer: 
-   :expand: records,versionedRelease, tender, award, tag
+   :jsonpointer:
+   :expand: records, versionedRelease, tag, tender, awards
    :title: versioned
 
 ```
 
 </div>
 
-## Merging rules
+## Merging specification
```

Comment:
Suggest reviewing structure of subsections here to make it easier to find information pertaining to specific types of release (compiled vs versioned).

Current subsections describing specific areas are good, and would be enhanced by a consistent structure ie

```markdown
### Objects
#### In a Compiled Release
....
#### In a Versioned Release
```

Reply:
I did notice that this is somewhat already present in some subsections. I think we could be more consistent.

Reply:
"Discarded fields" is the only section in which both types of releases are mentioned but not clearly segmented using indented lists. Fixed in 50ee7f9